printUserInfo() cause panic in OpenShift container

```
$ oc logs core-3-gpfsu -c core
panic: Failed to get current user: user: unknown userid 1000140000

goroutine 1 [running]:
panic(0x873060, 0xc8202c3a40)
        /home/aslak/.gvm/gos/go1.6.2/src/runtime/panic.go:481 +0x3e6
main.printUserInfo()
        /home/aslak/.gvm/pkgsets/go1.6.2/almighty/src/github.com/almighty/almighty-core/main.go:123 +0xe4
main.main()
        /home/aslak/.gvm/pkgsets/go1.6.2/almighty/src/github.com/almighty/almighty-core/main.go:38 +0x42
```

--- a/main.go
+++ b/main.go
@@ -120,14 +120,16 @@ func main() {
 func printUserInfo() {
 	u, err := user.Current()
 	if err != nil {
-		panic("Failed to get current user: " + err.Error())
-	}
-	fmt.Printf("Running as user name \"%s\" with UID %s.\n", u.Username, u.Uid)
-	/*
+		fmt.Printf("Failed to get current user: %s", err.Error())
+	} else {
+		fmt.Printf("Running as user name \"%s\" with UID %s.\n", u.Username, u.Uid)
+		/*
 		g, err := user.LookupGroupId(u.Gid)
 		if err != nil {
-			panic("Failed to lookup group: " + err.Error())
+			fmt.Printf("Failed to lookup group: %", err.Error())
+		} else {
+			fmt.Printf("Running with group \"%s\" with GID %s.\n", g.Name, g.Gid)
 		}
-		fmt.Printf("Running with group \"%s\" with GID %s.\n", g.Name, g.Gid)
-	*/
+		*/
+	}
 }

